xonsh/platform.py

```diff
@@ -212,26 +211,33 @@ def scandir(path):
                 WINDOWS_BASH_COMMAND = bashcmd
 
 #
-# Bash completions defaults
+# Environment variables defaults
 #
 
 BASH_COMPLETIONS_DEFAULT = ()
-""" A possibly empty tuple with default paths to Bash completions known for
-    the current platform.
+"""A possibly empty tuple with default paths to Bash completions known for
+the current platform.
 """
 
 if LINUX_DISTRO == 'arch':
     BASH_COMPLETIONS_DEFAULT = (
         '/etc/bash_completion',
         '/usr/share/bash-completion/completions')
+    PATH_DEFAULT = (os.path.expanduser('~/bin'), '/usr/local/sbin',
```

Comment:
I think for Arch and it's derivatives you can remove `~/bin` from the default `PATH`.  Although it doesn't do any harm being there, but the arch build tools are such that most people will install things into `/usr`